.github/workflows/python.yml

```diff
@@ -63,6 +63,13 @@ jobs:
           toolchain: stable
           components: rustfmt, clippy
 
+      - name: Install Python
```

Comment:
Moved this up because Ubuntu 18.04 comes with Python 3.6 and now we require Python 3.7.

Reply:
No it wasn't, see this build: https://github.com/huggingface/tokenizers/runs/5624656581?check_suite_focus=true

Reply:
Ahhh clippy is triggering it and it doesn't work after the upgrade !

So clippy was running against 3.6 Python interpreter, which is fine, since clippy wasn't running any python code anyway.

Reply:
Actually clippy will trigger running build scripts, which in turn runs python code from pyo3-ffi: https://github.com/PyO3/pyo3/blob/main/pyo3-ffi/build.rs